cmake_modules/FindSndfile.cmake

```diff
@@ -25,14 +25,19 @@ elseif (APPLE)
 	# TODO on non-apple platforms, we need to be able to test for >=1018.
 	# (On apple it is known true, because we bundle a later version.)
 	add_definitions("-DLIBSNDFILE_1018")
-
+elseif (WIN32)
+	message(STATUS "Using external_library libsndfile_win64")
+	set(SNDFILE_FOUND TRUE)
+	set(SNDFILE_INCLUDE_DIR ${CMAKE_SOURCE_DIR}/external_libraries/libsndfile_win64/include/)
+	set(SNDFILE_LIBRARIES ${CMAKE_SOURCE_DIR}/external_libraries/libsndfile_win64/libsndfile.lib)
+	
 else()
 	find_path(SNDFILE_INCLUDE_DIR sndfile.h
-		PATHS "$ENV{ProgramFiles}/Mega-Nerd/libsndfile/include"
+		PATHS "C:/Program Files/Mega-Nerd/libsndfile/include"
```

Comment:
I was flailing around, trying to get cmake to find my stuff. 

Cmake is picking the wrong environment variable.

PROGRAMFILES=C:\Program Files\
PROGRAMFILES(X86)=C:\Program Files (x86)\

i guess the brackets confuse cmake and it thinks PROGRAMFILES=C:\Program Files (x86)\

which then links the 32bit library and badness ensues.